textworld/core.py

```diff
@@ -274,9 +275,18 @@ def __call__(self, env: Environment) -> Environment:
         self._wrapped_env = env
         return self
 
+    def __getattr__(self, attr: str):
+        return getattr(self._wrapped_env, attr)
```

Comment:
This seems somewhat risky to me.  At the very least we should guard it from when `_wrapped_env` is `None`.

Reply:
The fallback could be
```
return super().__getattr__(self, attr)
```

Reply:
Do you mean the following?
```python
return super().__getattribute__(attr)
```

Reply:
```python
    def __getattr__(self, attr: str):
        if self._wrapped_env:
            return getattr(self._wrapped_env, attr)

        return super().__getattribute__(attr)
```